src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -230,7 +230,7 @@ public void CollectionObjectsPropertiesNamingValidation()
         public void BodyTopLevelPropertiesValidation()
         {
             var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "body-top-level-properties.json"));
-            messages.AssertOnlyValidationMessage(typeof(BodyTopLevelProperties), 2);
+            messages.AssertOnlyValidationMessage(typeof(BodyTopLevelProperties), 1);
```

Comment:
The count went down because we are using a different definition of a resource, the earlier definition considered models named ```Resource``` as resource models too but in the new definition we disregard models that are named as ```Resource```, ```TrackedResource``` or ```ProxyResource```